config/kraft/README.md

```diff
@@ -93,7 +96,7 @@ controller.quorum.voters=1@controller1.example.com:9093,2@controller2.example.co
 
 Each broker and each controller must set `controller.quorum.voters`.  Note that the node ID supplied in the `controller.quorum.voters` configuration must match that supplied to the server.
 So on controller1, node.id must be set to 1, and so forth.  Note that there is no requirement for controller IDs to start at 0 or 1.  However, the easiest and least confusing way to allocate
-node IDs is probably just to give each server a numeric ID, starting from 0.
+node IDs is probably just to give each server a numeric ID, starting from 0.  Also note that the node ID is global and can not conflict with other node ID no matter what process role is.
```

Comment:
Add a sentence to mention the node ID is global and can't conflict with each other no matter what process role is.